src/main/java/com/example/exercise2/repository/entity/BookEntity.java

```diff
@@ -0,0 +1,96 @@
+package com.example.exercise2.repository.entity;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@Entity
+@EntityListeners(AuditingEntityListener.class)
+public class BookEntity extends BaseEntity {
+
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
You can set this the strategy here and in Author as IDENTITY, such that id of books can also start at 1, regardless if created multiple authors before adding first book to author 1.

@id
@GeneratedValue(strategy = GenerationType.IDENTITY)